#goal definition
string frame
float32 gripper_open
float32 gripper_closed
geometry_msgs/Pose pickup_pose
geometry_msgs/Pose draw_pose
string topic
---
#result definition
---
#feedback
